Cargo.toml

```diff
@@ -22,4 +22,4 @@ features = ["extension-module"]
 
 [package.metadata.maturin]
 requires-python = ">=3.7"
-project-url =  { Source = "https://github.com/quickwit-inc/tantivy-py" }
+project-url =  ["https://github.com/quickwit-inc/tantivy-py"]
```

Comment:
afaics this change breaks building with maturin tooling

```
❯ maturin build
💥 maturin failed
  Caused by: Failed to parse Cargo.toml at Cargo.toml
  Caused by: invalid type: sequence, expected a map for key `package.metadata.maturin.project-url` at line 23 column 1
```

Reply:
@jgreene Can you confirm and fix?